learn/advanced/updating.md

```diff
@@ -20,6 +20,10 @@ This guide only works for versions v0.15 and above. If you are using an older ve
 
 This section contains instructions for upgrading from specific versions. Most versions don't require version-specific steps and you should be able to upgrade directly. If the version you are upgrading from isn't listed here, no additional steps are required.
 
+- If you are updating **from v0.19 or below**, please ensure all updates finish processing before creating the dump. `enqueued` updates will not be exported and may result in data loss.
+
+- If you are updating from **v0.20 or below**, the database creation date will not be imported to the new version.
```

Comment:
My opinion is that this is not important enough to users to document here.

For one thing, it only affects users that are ten or more versions behind the latest release. More importantly, I personally cannot imagine any situation where losing the database creation date would have an impact on existing code processes.

Since it's distracting in a guide that is already very information and warning-heavy, I am suggesting we remove it. @dureuill Do you object?

Reply:
No objection from here